### ISSUE TITLE
ModuleNotFoundError: No module named '_curses'

### ISSUE THREAD

**Issue by Vxer-Lee** [0cyn/ktool#43]
OS: Microsoft Windows 10 x64

An error occurred after running Ktool.
```python
Traceback (most recent call last):
  File "c:\python3\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\python3\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\python3\scripts\ktool.exe\__main__.py", line 4, in <module>
  File "c:\python3\lib\site-packages\ktool\ktool_script.py", line 54, in <module>
    from ktool.window import KToolScreen, external_hard_fault_teardown
  File "c:\python3\lib\site-packages\ktool\window.py", line 31, in <module>
    import curses
  File "c:\python3\lib\curses\__init__.py", line 13, in <module>
    from _curses import *
ModuleNotFoundError: No module named '_curses'
```
![image](https://user-images.githubusercontent.com/27657137/175235378-09b1847e-0903-4059-b9ad-2e1571dc40f9.png)

**Comment by cxnder** [comment]
Install curses on windows with `python -m pip install windows-curses`. It should work then.

Issue is now tracking a better failure state for this on win.

**Comment by 0cyn** [comment]
Resolved with d57000a916b9043f028bbe06b4974ebb1a147a1d merger of #48
